framework/mesh/unpartitioned_mesh/unpartitioned_mesh.h

```diff
@@ -191,23 +124,21 @@ class UnpartitionedMesh
                      int cell_material_id,
                      const std::vector<std::vector<uint64_t>>& proxy_faces);
 
-  ~UnpartitionedMesh();
+  void CleanUp();
 
-  void CleanUp()
-  {
-    for (auto& cell : raw_cells_)
-      delete cell;
-    for (auto& cell : raw_boundary_cells_)
-      delete cell;
-    vertices_.clear();
-    vertices_.shrink_to_fit();
-    raw_cells_.clear();
-    raw_cells_.shrink_to_fit();
-    raw_boundary_cells_.clear();
-    raw_boundary_cells_.shrink_to_fit();
-    vertex_cell_subscriptions_.clear();
-    vertex_cell_subscriptions_.shrink_to_fit();
-  }
+protected:
+  std::vector<Vertex> vertices_;
+  std::vector<LightWeightCell*> raw_cells_;
+  std::vector<LightWeightCell*> raw_boundary_cells_;
+  std::vector<std::set<uint64_t>> vertex_cell_subscriptions_;
+
+  /// Spatial mesh dimension
+  unsigned int dim_;
+  struct OrthoMeshAttributes ortho_attrs_;
+  MeshAttributes attributes_ = NONE;
+  Options mesh_options_;
+  BoundBox bound_box_;
+  std::map<uint64_t, std::string> boundary_id_map_;
```

Comment:
```suggestion
  unsigned int dim_;  ///< Spatial dimension
  struct OrthoMeshAttributes ortho_attrs_;
  MeshAttributes attributes_ = NONE;
  Options mesh_options_;
  BoundBox bound_box_;
  std::map<uint64_t, std::string> boundary_id_map_;
  
  std::vector<Vertex> vertices_;
  std::vector<LightWeightCell*> raw_cells_;
  std::vector<LightWeightCell*> raw_boundary_cells_;
  std::vector<std::set<uint64_t>> vertex_cell_subscriptions_;
```

Reply:
Should it be `struct OrthoMeshAttributes` or just `OrthoMeshAttributes`?

Reply:
`struct` is not needed, I will remove it...